src/runtime/step_runner.js

```diff
@@ -8,22 +8,30 @@ const {beginTiming, endTiming} = Time
 
 async function run({attachmentManager, defaultTimeout, scenarioResult, step, stepDefinition, parameterRegistry, world}) {
   beginTiming()
-  const parameters = stepDefinition.getInvocationParameters({scenarioResult, step, parameterRegistry})
-  const timeoutInMilliseconds = stepDefinition.options.timeout || defaultTimeout
-
-  let error, result
-  const validCodeLengths = stepDefinition.getValidCodeLengths(parameters)
-  if (_.includes(validCodeLengths, stepDefinition.code.length)) {
-    const data = await UserCodeRunner.run({
-      argsArray: parameters,
-      fn: stepDefinition.code,
-      thisArg: world,
-      timeoutInMilliseconds
-    })
-    error = data.error
-    result = data.result
-  } else {
-    error = stepDefinition.getInvalidCodeLengthMessage(parameters)
+  let error, result, parameters
+
+  try {
+    parameters = stepDefinition.getInvocationParameters({scenarioResult, step, parameterRegistry})
+  } catch(err) {
+    error = err
+  }
+
+  if(!error) {
```

Comment:
I agree a linter would be ideal here but I don't really see the different between a linter and me asking for a whitespace change.

---

> But even more important is to display appreciation for contributions.

Does me saying "Awesome @aslakhellesoy!" earlier in this PR not count? Anytime I merge a PR, I always have a "Thank you" comment after and try to comment on the PR letting the user know when the change has been released.

---

> We've all got our pet peeves, I certainly have mine.

One of mine is to actually have all the keys of an object sorted. This was something that got messed up in a previous PR. That I did not think was worth asking you to do.

---

> If minor formatting issues like this bugs you I suggest you tweak the linting rules, or write your own if you can't find rules that are strict enough for your liking.

I have done this in the past for other issues and if this becomes a common problem I would certainly look into it. But I haven't really run into this much as the general styling is all I'm asking. A line of whitespace between functions. A space between an if and the paren.

---

Thank you for the PRs. I'll take care of merging them and dealing the my comments